### PR TITLE
Added docstrings D102 to public method on Bio.Wise

### PR DESCRIPTION
**Pull request by svalqui**
This pull request addresses issue #1963 

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file and understand that AppVeyor and
TravisCI will be used to confirm the Biopython unit tests and ``flake8`` style
checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

File: Bio/Wise/dnal.py [hunk 1]
```diff
@@ -116,6 +117,7 @@ def __str__(self):
 
 
 def align(pair, match=_SCORE_MATCH, mismatch=_SCORE_MISMATCH, gap=_SCORE_GAP_START, extension=_SCORE_GAP_EXTENSION, **keywds):
```
Comment (MarkusPiotrowski) [inline Bio/Wise/dnal.py]: Can you break the line after ```extension=_SCORE_GAP_EXTENSION```? (To make it <80 char; we do not enforce this yet)

File: Bio/Wise/psw.py [hunk 2]
```diff
@@ -133,7 +143,7 @@ def align(pair,
           gap_start=None,
           gap_extension=None,
           *args, **keywds):
```
Comment (MarkusPiotrowski) [inline Bio/Wise/psw.py]: This may be a matter of taste, but I would put all the arguments in one or two lines (not exceeding 79 characters).

Comment (peterjc) [reply Bio/Wise/psw.py]: Largely personal taste, and we have no compelling reason to change this as part of this pull request.

(Separately I am becoming a fan of the code formatting tool ``black`` which would decide this kind of thing automatically)

File: Bio/Wise/dnal.py (hunk 1 above)
Comment (peterjc) [reply Bio/Wise/dnal.py]: That would be nice, but isn't essential for this pull request focused on the docstrings.

**Comment by peterjc** [comment]
Looks good - Markus is right there are some unrelated style issues we could work on, but nothing which blocks merging your docstring work. Thank you!

**Comment by MarkusPiotrowski** [comment]
A different discussion and yes, offen a matter of personal taste, but I hate what ```black``` would do to with expressions in matching brackets, e.g.:

```
# in:

ImportantClass.important_method(exc, limit, lookup_lines, capture_locals, extra_argument)

# out:

ImportantClass.important_method(
    exc, limit, lookup_lines, capture_locals, extra_argument
)
```
And I find it even more terrible, what it would do to function arguments:
```
# out:

def very_important_function(
    template: str,
    *variables,
    file: os.PathLike,
    engine: str,
    header: bool = True,
    debug: bool = False,
):
    """Applies `variables` to the `template` and writes to `file`."""
    with open(file, "w") as f:
        ...
```

**Comment by peterjc** [comment]
As you say a matter of taste - I had some niggles with black's style, I still don't agree with all the choices, but it has grown on me - and delegating this is liberating (both on solo projects, and likely more so on group projects).  I've filed #2008 as a more visible place to have this discussion.

**Comment by svalqui** [comment]
@MarkusPiotrowski , My apologies, I couldn't include your feedback on time I will consider for future PR.
This PR had take me more than expected as I had difficulties installing wise 2.4.1, to give me a better sense of what the module does, It has not work on Ubuntu 18.04;  wise 2.4.1 seems stale, so I am not sure about the future of Bio.Wise.
Cheers,

**Comment by MarkusPiotrowski** [comment]
@svalqui No problem, everything's fine! I know it's often hard to add meaninful docstrings, because you need to understand what the code is really doing. I also made this experience several times!

**Pull request merged**